### ISSUE TITLE
Contribution to SpriterMC

### ISSUE THREAD

**Issue by wastedabuser** [SammyJoeOsborne/SpriterMC#2]
I am using ASC 2.0.
I've fixed those and I would like to contribute to this project with other improvements of mine!

So far i have fixed the following issues:
- Changing image texture does not seem to work (prevFileDirty and nextFileDirty);
- Setting texture with different size results in stretching the texture to the original image size;
- Playing the animation in reverse starts the sequence with the textures from the first key frame rather than the last.
- Pivot points are parsed from the keys, but it seems they may appear on the file nodes.

Please, contact me! :)

**Comment by aureliendc** [comment]
Hi @wastedabuser!
Nice! I'm really interrested by your fixes :) Could you share them through a fork ?
Thanks in advance

**Comment by wastedabuser** [comment]
Yes, i just pushed my changes to a fork under my account. Unfortunately the line endings conversion messed things a bit :(

**Comment by aureliendc** [comment]
Thanks! No problem, I will try to identicate your changes by reconverting the files.

**Comment by aureliendc** [comment]
It works :) Thank you @wastedabuser !
Do you matter if I split your changes into several commits and create a pull request ?

**Comment by wastedabuser** [comment]
Well, no problem.... but I wonder why do you need that?

**Comment by wastedabuser** [comment]
@aureliendc, sorry for my later reply, i checked your commits and it seems it's not going to be a straight forward merge :( There are a lot of conflicts...
